net/tcp/tcp_listen.c

```diff
@@ -100,9 +104,35 @@ FAR struct tcp_conn_s *tcp_findlistener(uint16_t portno)
       if (conn && conn->lport == portno)
 #endif
         {
-          /* Yes.. we found a listener on this port */
+#ifdef CONFIG_NET_IPv6
+#  ifdef CONFIG_NET_IPv4
+          if (domain == PF_INET6)
+#  endif
+            {
+              if (net_ipv6addr_cmp(conn->u.ipv6.laddr, uaddr->ipv6.laddr) ||
```

Comment:
CI is broken due to missing `string.h` include

Reply:
Yes, the header file which provided net_ipv6addr_cmp macro should include string.h to make self contained.

Reply:
Thanks, let me fix it